arch/arm64/boot/dts/freescale/crocodile.dtsi

```diff
@@ -168,71 +167,71 @@
 			// VDD_DRAM&PU_0V9
 			buck3_reg: BUCK3 {
 				regulator-name = "BUCK3";
-				regulator-min-microvolt = <805000>;
-				regulator-max-microvolt = <1000000>;
+				regulator-min-microvolt = <850000>;
+				regulator-max-microvolt = <950000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// VDD_3V3/NVCC_3V3
 			buck4_reg: BUCK4 {
 				regulator-name = "BUCK4";
-				regulator-min-microvolt = <3000000>;
-				regulator-max-microvolt = <3600000>;
+				regulator-min-microvolt = <3300000>;
+				regulator-max-microvolt = <3300000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// VDD_1V8/NVCC_1V8
 			buck5_reg: BUCK5 {
 				regulator-name = "BUCK5";
-				regulator-min-microvolt = <1650000>;
-				regulator-max-microvolt = <1950000>;
+				regulator-min-microvolt = <1800000>;
+				regulator-max-microvolt = <1800000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// NVCC_DRAM_1V1
 			buck6_reg: BUCK6 {
 				regulator-name = "BUCK6";
-				regulator-min-microvolt = <1060000>;
-				regulator-max-microvolt = <1140000>;
+				regulator-min-microvolt = <1100000>;
+				regulator-max-microvolt = <1100000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// NVCC_SNVS_1V8
 			ldo1_reg: LDO1 {
 				regulator-name = "LDO1";
-				regulator-min-microvolt = <1620000>;
-				regulator-max-microvolt = <1980000>;
+				regulator-min-microvolt = <1800000>;
+				regulator-max-microvolt = <1800000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// VDD_SNVS_0V8
 			ldo2_reg: LDO2 {
 				regulator-name = "LDO2";
-				regulator-min-microvolt = <760000>;
-				regulator-max-microvolt = <900000>;
+				regulator-min-microvolt = <800000>;
+				regulator-max-microvolt = <800000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// VDDA_1V8
 			ldo3_reg: LDO3 {
 				regulator-name = "LDO3";
-				regulator-min-microvolt = <1710000>;
-				regulator-max-microvolt = <1890000>;
+				regulator-min-microvolt = <1800000>;
+				regulator-max-microvolt = <1800000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
 
 			// VDD_PHY_0V9
 			ldo4_reg: LDO4 {
 				regulator-name = "LDO4";
-				regulator-min-microvolt = <855000>;
-				regulator-max-microvolt = <1000000>;
+				regulator-min-microvolt = <900000>;
+				regulator-max-microvolt = <900000>;
 				regulator-boot-on;
 				regulator-always-on;
 			};
```

Comment:
You need to adapt the LD05 as well to 1.8/3.3V since these values are the only min/max values for SD2

Reply:
/sys/kernel/debug/regulator/regulator_summary says LDO5=1800mV - so i just narrow it down to that(?)

Reply:
Please set it to 1.8V and 3.3V since SD card speed negotiation will always start with slowest speed which is based on 3.3V levels.

Reply:
aha! thanks for the background/explanation - ACK